pom.xml

```diff
@@ -23,7 +23,7 @@
         </developer>
     </developers>
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/clover-plugin.git</connection>
+        <connection>scm:git:https://github.com/jenkinsci/clover-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/clover-plugin.git</developerConnection>
```

Comment:
Add SSH here too?

Reply:
The "short form" (`git@github.com`) seems to be more commonly used in Jenkins repositories, though either is correct and has the same result as far as I can tell.  I'm happy to change it to `ssh://` form if you prefer that.

Reply:
I'd prefer `scm:git:ssh://git@github.com:jenkinsci/clover-plugin.git`

Reply:
Thanks.  Changed in beb0c93